### ISSUE TITLE
[BUG] Unit test for Huobi market not running

### ISSUE THREAD

**Issue by dennisocana** [CoinAlpha/hummingbot#1207]
**Describe the bug**

I keep getting a blinking cursor only when running the unit test for `test_huobi_market.py`. API and secret keys are configured in `config_local.py`. See result from `python test_huobi_market.py`

[python_test_huobi_market.log](https://github.com/CoinAlpha/hummingbot/files/3977681/python_test_huobi_market.log)

As per @veecos2 this is due to config encryption feature that was released recently.

**Comment by veecos2** [comment]
@dennisocana I am not sure why you are unable to run nosetests. I am able to run them for Huobi. Can you make a new instance and see if you can run it there ?.

**Comment by dennisocana** [comment]
@veecos2 I did a clean reinstall of my Linux subsystem in Windows and I'm still getting the same problem for running the Huobi market test in the updated development branch.

To eliminate the possibility of my computer having problems, I created an AWS EC2 instance running in Ubuntu 18.04 LTS --> installed Hummingbot from source --> switched to development branch --> installed and compiled --> same problem.

To check if it's an API key problem, I used other keys from two other accounts with all the permissions enable --> same problem.

In my `config_local.py`

```
huobi_api_key = "myapikey"
huobi_api_secret_key = "mysecretkey"
```
